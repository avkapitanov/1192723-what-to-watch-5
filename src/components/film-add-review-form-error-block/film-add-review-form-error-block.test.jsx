import React from "react";
import renderer from "react-test-renderer";

import FilmAddReviewFormErrorBlock from "./film-add-review-form-error-block";

const errorText = `Something wrong, please, try again later`;

it(`FilmAddReviewFormErrorBlock component render correctly`, () => {
  const tree = renderer.create(
      <FilmAddReviewFormErrorBlock
        errorText={errorText}
      />
  ).toJSON();

  expect(tree).toMatchSnapshot();
});


