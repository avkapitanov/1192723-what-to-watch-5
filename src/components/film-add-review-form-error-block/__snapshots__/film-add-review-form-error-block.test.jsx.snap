// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`FilmAddReviewFormErrorBlock component render correctly 1`] = `
<div
  style={
    Object {
      "color": "red",
      "paddingTop": "10px",
      "textAlign": "center",
    }
  }
>
  Something wrong, please, try again later
</div>
`;
